docs/tutorial_volume_annotation.md

```diff
@@ -26,14 +26,22 @@ Create a new segment by selecting a new segment ID. Use the trace tool to draw m
 ![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/03_new_segments_lasso.mp4){: autoplay loop muted}
 
 ## Smart Annotation Tools
+Now, let’s explore the smart tools of WEBKNOSSOS. Choose the quick-select tool and turn on the AI option. Draw a rectangle around the cell you want to segment. Create and annotate new segments by pressing the keyboard shortcut “C”.
 
-Now, let’s explore the smart tools of WEBKNOSSOS. Choose the quick-select tool and draw a rectangle around the cell you want to segment. Scroll to the next slice and repeat.
+![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/04_new_AI_quick_select.mp4){: autoplay loop muted}
 
-Annotating a nucleus with the quick-select tool.
-Next, jump a few more slices in the Z direction. Annotate your cell again. Click the “interpolation” tool and congratulations! WEBKNOSSOS just segmented each slice in between the two and saved you time. Repeat this super fast workflow as much as you need.
+Quick tip: When dealing with long and complex shapes, try drawing multiple rectangles on different areas. The annotations will merge and your complex shape will be segmented in no time.
+
+![type:video](https://static.webknossos.org/assets/docs/tutorial-volume-annotation/04_new_tip_long_cell.mp4){: autoplay loop muted}
+
+Now, annotate a new segment. Click the “interpolation” tool and congratulations! WEBKNOSSOS just segmented each slice in between the two and saved you time.
```

Comment:
```suggestion
Now, annotate a segment on a new slice. Skip a few sections forward and annotate again. To interpolate the two sections, click the “interpolation” tool. All the slices in between the two original ones have now been automatically annotated by WEBKNOSSOS.
```

This was changed today, because the original description was misleading.